frontend/src/components/common/Grid/components/Loader/index.tsx

```diff
@@ -1,11 +1,18 @@
 import React from "react";
 import { LoadingIndicator } from "@czi-sds/components";
-import { Loader as GridLoader } from "./style";
 
-export default function Loader(): JSX.Element {
+interface Props {
+  className?: string;
+  sdsStyle?: "minimal" | "tag";
+}
+
+export default function Loader({
+  className,
```

Comment:
non blocking discussion: should we just spread the `...rest` prop onto the `<div>`, so it gets any other props passed from the parent, including `className` and potentially `data-testid` in the future if needed?

E.g.,

```tsx
function Loader({sdsStyle, ...rest}) {
  <div {...rest}>
    <LoadingIndicator sdsStyle={sdsStyle} />
  </div>
}
```